CONTRIBUTING.md

```diff
@@ -158,6 +158,23 @@ To ensure that our documentation adheres to the [OpenSearch Project Style Guidel
 
 Optionally, you can install the [Vale VSCode](https://github.com/chrischinchilla/vale-vscode) extension, which integrates Vale with Visual Studio Code. By default, only _errors_ and _warnings_ are underlined. To change the minimum alert level to include _suggestions_, go to **Vale VSCode** > **Extension Settings** and select **suggestion** in the **Vale > Vale CLI: Min Alert Level** dropdown list. 
 
+## Troubleshooting
+
+This section details potential solutions for known issues.
+
+### Installing Ruby on Apple Silicon machines
+
+If you're having trouble installing Ruby with `rvm` on Apple Silicon machines, it could be to do an OpenSSL version misalignment. To fix this issue, use the following command, replacing `<openssl-version>` with your [desired version](https://github.com/ruby/openssl/blob/master/README.md):
```

Comment:
```suggestion
If you're having trouble installing Ruby with `rvm` on an Apple Silicon machine, it could be because of an OpenSSL version misalignment. To fix this issue, use the following command, replacing `<openssl-version>` with your [desired version](https://github.com/ruby/openssl/blob/master/README.md):
```